modules/test/playwright/pages/dynamic-data-mapping-form-web/FormBuilderPage.ts

```diff
@@ -11,6 +11,7 @@ import {FormsPage} from './FormsPage';
 export class FormBuilderPage {
 	readonly copyButton: Locator;
 	readonly entriesTab: Locator;
+	readonly formDescription: Locator;
```

Comment:
are these changes really needed? we cannot avoid adding the description? (just to avoid - if possible - touching other team files)